src/components/common/NetworkSelector/NetworkMultiSelector.tsx

```diff
@@ -0,0 +1,76 @@
+import useChains from '@/hooks/useChains'
+import { type ReactElement, useState } from 'react'
+import { Checkbox, Autocomplete, TextField, Chip } from '@mui/material'
+import type { ChainInfo } from '@safe-global/safe-gateway-typescript-sdk'
+import ChainIndicator from '../ChainIndicator'
+import { useRouter } from 'next/router'
+import useWallet from '@/hooks/wallets/useWallet'
+import { getNetworkLink } from '.'
+import css from './styles.module.css'
+
+const NetworkMultiSelector = ({
+  selectedNetworks,
+  setSelectedNetworks,
+}: {
+  selectedNetworks: ChainInfo[]
+  setSelectedNetworks: (selectedChains: ChainInfo[]) => void
+}): ReactElement => {
+  const { configs } = useChains()
```

Comment:
We cannot offer all chains.

We have to make sure that the chains have matching contracts deployed.
Those are all chains that have a canonical 1.4.1 deployment.

Reply:
We still need to offer a way to deploy Safes on networks that do not support 1.4.1 yet though. 
Maybe we could make it dependent on the first selected network and disable all networks that are not compatible.

Reply:
I made the changes to the options here, can you check it again and make sure it behaves as expected?